apps/gitness/src/pages-v2/webhooks/webhook-list.tsx

```diff
@@ -78,7 +106,26 @@ export default function WebhookListPage() {
 
   useEffect(() => {
     setQueryPage(page)
+
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [page, queryPage, setPage])
-  return <RepoWebhookListPage useWebhookStore={useWebhookStore} useTranslationStore={useTranslationStore} />
+
+  return (
+    <div>
```

Comment:
```suggestion
    <>
```
I think we can just use a Fragment here.